packages/cyberstorm/src/components/MetaItem.tsx

```diff
@@ -12,7 +12,7 @@ export interface MetaItemProps {
  * Used for displaying a single data point (e.g. an amount
  * of likes or a size of a package) with an icon next to it
  */
-export const MetaItem: React.FC<MetaItemProps> = (props) => {
+export const MetaItem: React.FC<MetaItemProps> = React.forwardRef((props) => {
```

Comment:
I think we're going a bit overboard with the refs here (or maybe I'm missing something)? Maybe hit me up on Discord? Anyways the ref isn't even used on this component.